dbms/src/BaseFile/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 PingCAP, Inc.
```

Comment:
I think removing this file does not affects anything

Reply:
`add_subdirectory` requires the subdirectory to contain a `CMakeLists.txt` file.

```
add_subdirectory (BaseFile)
```

Reply:
But actually we don't need to add the `add_subdirectory` statement

Reply:
yeah, but if we need to split into smaller compile units, we need to do this, I think we just keep this.